__tests__/app.test.js

```diff
@@ -55,7 +55,7 @@ describe('app routes', () => {
     ];
 
     const data = await fakeRequest(app)
-      .get('/animals')
+      .get('/coffees')
```

Comment:
nice job, also nice touch changing your users to fit your data structure and user narrative